src/iocore/cache/StripeSM.h

```diff
@@ -168,7 +168,7 @@ class StripeSM : public Continuation, public Stripe
    *
    * @see START_POS
    */
-  StripeSM(CacheDisk *disk, off_t blocks, off_t dir_skip, int avg_obj_size = -1);
+  StripeSM(CacheDisk *disk, off_t blocks, off_t dir_skip, int avg_obj_size = -1, int fragment_size = -1);
```

Comment:
Looks like the docstring is out of date.

Reply:
Updated.